tests/copilot_helper/fixtures/make_addons/expected/environments/addons/my-aurora-db.yml

```diff
@@ -44,6 +44,10 @@ Mappings:
       DBMinCapacity: 0.5
       DBMaxCapacity: 8
       DeletionPolicy: Delete
+      SnapshotIdentifier: my-recent-snapshot
+
+Conditions:
+  UseDBSnapshot: !Not [!Equals [!FindInMap [myAuroraDbEnvironmentConfigMap, !Ref Env, SnapshotIdentifier], ""]]
```

Comment:
I'm wondering if these conditionals need to be prefixed with the `myAuroraDb` bit, as the addons are merged into one template before being deployed. This could mean that the conditionals override each other. Worth testing to confirm in demodjango where we have both RDS and Aurora.

This is probably fine for now without the straight RDS template in place, but may as well address up front.

Reply:
No, hadn't groked what you were saying. Thought you meant that it would be the same as the postgresRDS DB version when we come to that, and that you couldn't have two conditionals in the same deployment the same name.  Fixed.